gate-core/src/main/groovy/com/netflix/spinnaker/gate/services/internal/ClouddriverService.groovy

```diff
@@ -257,7 +257,10 @@ interface ClouddriverService {
   Map getSecurityGroups()
 
   @GET('/securityGroups/{account}/{type}')
-  Map getSecurityGroups(@Path("account") String account, @Path("type") String type, @Query("region") String region)
+  Map getSecurityGroups(@Path("account") String account, @Path("type") String type)
+
+  @GET('/securityGroups/{account}/{type}')
```

Comment:
Different path here than for getSecurityGroups?

Reply:
Same path, but with a query param (Clouddriver's API is not designed very well)